docs/cheatsheet/cheatsheet.html

```diff
@@ -31,13 +31,14 @@
 <div class="header">
 <div class="usage">
 <span class="one">EarlGrey()</span>.selectElementWithMatcher(<span class="two">ViewMatcher</span>)
+                     .indexOf(<span class="three">index</span>)
```

Comment:
Um, I don't think this is needed. If so, we should be adding usingSearchAction etc. as well.